pkg/client/client.go

```diff
@@ -0,0 +1,107 @@
+package client
```

Comment:
There should probably be some copyright stuff here.

Reply:
Fixed